spark-commons/src/test/scala/za/co/absa/spark/commons/errorhandling/implementations/ErrorMessageArrayTest.scala

```diff
@@ -197,6 +193,41 @@ class ErrorMessageArrayTest extends AnyFunSuite with SparkTestBase {
     val result = resultDfToResult(resultDf)
 
     assert(result == expected)
+  }
+
+  test("errorColumnType should return errCol type schema") {
```

Comment:
I think that this test can be dropped, it doesn't actually do anything apart from copying the same schema as the underlying function `errorColumnType`.

But I like the next test, so I think that you don't need to implement any other tests here in this module :)